mmv1/products/dataplex/Entry.yaml

```diff
@@ -0,0 +1,276 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'Entry'
```

Comment:
Since this is a child resource of Entry Group it really should be called "EntryGroupEntry" (which is a mouthful, but it matches the usual TPG naming scheme)